ui/src/reusable_ui/components/card_select/CardSelectList.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {PureComponent, ReactElement} from 'react'
+import {ErrorHandling} from 'src/shared/decorators/errors'
+
+import {CardSelectCardProps} from 'src/types/cardSelect'
+
+interface Props {
+  children: Array<ReactElement<CardSelectCardProps>>
```

Comment:
Whoa wild, does this throw an error if the dev tries to pass in children that are not `CardSelectCard`?

Reply:
Yes, this throws a typescript error if children are passed that are not CardSelectCard type.